app/models/bookmark.rb

```diff
@@ -24,10 +24,12 @@ class Bookmark < Content
   belongs_to :owner, class_name: 'User'
 
   validates :title,     presence: { message: "Le titre est obligatoire" },
-                        length: { maximum: 100, message: "Le titre est trop long" }
+                        length: { maximum: 100, message: "Le titre est trop long" },
```

Comment:
Problem detected by a user: same message "Le titre est trop long" for a news title and a link title in the news... Misleading to find where you need to change the title.